db/warehouse/migrate/20161111214343_add_report_service_view.rb

```diff
@@ -1,10 +1,16 @@
 class AddReportServiceView < ActiveRecord::Migration
   def up
     model = GrdaWarehouse::Hud::Service
```

Comment:
I think this needs the faked enrollment_id too.  These only make sense in the context of an enrollment.